static/app/components/nav/overlay.tsx

```diff
@@ -0,0 +1,60 @@
+import {type ComponentProps, forwardRef} from 'react';
+import styled from '@emotion/styled';
+
+import InteractionStateLayer from 'sentry/components/interactionStateLayer';
+import Link from 'sentry/components/links/link';
+import {Overlay} from 'sentry/components/overlay';
+import {space} from 'sentry/styles/space';
+
+export const OverlayMenu = forwardRef<
```

Comment:
I'm scarred from all the generic naming we have in the codebase, but how about?
```suggestion
export const NavOverlayMenu = forwardRef<
```

Reply:
Point taken! File (and all new components) removed in f1efc6c6948